### PR TITLE
Recovery password in api

### PR DESCRIPTION
**Pull request by LPMendoza**
# Description
The recover password section was added to the web page and for the api that app mobile uses.
The user can go to the recover section in the web page and type his email to receive an email with his new password, and for the mobile app a endpoint has been enabled for the same.

- Run the project.
- go to http://locahost:3000/
- Click on Recover password.
- Type your email.
- Check for the email with your new password in your account.
 
## Web recover
image
image
image
image

## API recover
This is the petition
image
This is the response you get
image

### CODE REVIEW

**Review by LDVera — APPROVED** [review]
all looks good only remember the blank line in the end of the rb files, good job 💯

File: config/database.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ development:
   database: db_covid_development
   host: localhost
   user: root
-  port: 5433
+  port: 5432
   password: toor
```
Comment (OchoaHub) [inline config/database.yml:19]: Ty for the password bro

**Review by OchoaHub — APPROVED** [review]
Nice API guys!

File: config/database.yml (hunk 1 above)
Comment (LPMendoza) [reply config/database.yml:19]: They are the password in my local bro

**Review by jnegrete15 — APPROVED** [review]

**Pull request merged**